app/Http/Controllers/QuestionApiController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Question;
+use Illuminate\Http\Request;
+
+class QuestionApiController extends Controller
+{
+    public function create(Request $request)
+    {
+        $request->validate([
+            'question' => 'required|string|min:5|max:70',
+            'hint' => 'nullable|string|min:5|max:70',
+            'points' => 'required|integer',
+            'quiz_id' => 'required|integer',
+        ]);
+
+        $newQuestion = new Question;
+
+        $newQuestion->question = $request->question;
+        $newQuestion->hint = $request->hint;
+        $newQuestion->points = $request->points;
+        $newQuestion->quiz_id = $request->quiz_id;
+
+        $newQuestion->save();
```

Comment:
Here it would be worth checking for a failure. This save method will return a boolean to tell you whether or not it worked.

This is where you should return the 500 with message from the docs.

You can just do:

```
if (! $newQuestion->save()) {
// return 500
}
```